src/controller/CHIPDeviceController.h

```diff
@@ -564,6 +564,21 @@ class DLL_EXPORT DeviceCommissioner : public DeviceController,
     void PerformCommissioningStep(DeviceProxy * device, CommissioningStage step, CommissioningParameters & params,
                                   CommissioningDelegate * delegate, EndpointId endpoint, Optional<System::Clock::Timeout> timeout);
 
+    /**
+     * @brief
+     *   This function validates the Attestation Information sent by the device.
+     *
+     * @param[in] attestationElements Attestation Elements TLV.
+     * @param[in] signature           Attestation signature generated for all the above fields + Attestation Challenge.
+     * @param[in] attestationNonce    Attestation nonce
+     * @param[in] pai                 PAI certificate
+     * @param[in] dac                 DAC certificates
+     * @param[in] proxy               device proxy that is being attested.
+     */
+    CHIP_ERROR ValidateAttestationInfo(const ByteSpan & attestationElements, const ByteSpan & signature,
+                                       const ByteSpan & attestationNonce, const ByteSpan & pai, const ByteSpan & dac,
+                                       DeviceProxy * proxy);
```

Comment:
This will need to be updated to take in more info, such as current vid/pid previously read from basic info cluster. Can be done as a follow-u[

Reply:
Basic read will be handled in a follow up. Not sure about the validation. Is there an issue for that open?